deel/puncc/api/utils.py

```diff
@@ -161,7 +162,7 @@ def alpha_calib_check(alpha: float, n: int, complement_check: bool = False):
 
         0 < \\alpha \cdot (1+1/n) < 1 \implies 0 < \\alpha < n/(n+1)
 
-    :param float alpha: target quantile order.
+    :param float or np.ndarray alpha: target quantile order.
```

Comment:
Let's use `Union` instead of `or` in the docstring:

`:param Union[float, np.array] alpha: target quantile order.`